src/frontend/index.tsx

```diff
@@ -32,6 +32,7 @@ if (process.env.NODE_ENV !== 'production') {
 
 // Last ned land-navn for statsborgeskap
 import(`i18n-iso-countries/langs/nb.json`).then(result => registerLocale(result));
+import(`i18n-iso-countries/langs/en.json`).then(result => registerLocale(result));
```

Comment:
Trenger ikke å fjerne, men kanskje legg inn en TODO på å fjerne engelsk her siden vi ikke har språktekster for dem enda.

Reply:
Det kan jeg gjøre :)